controller/hybridgateway/converter/http_route.go

```diff
@@ -450,8 +448,7 @@ func (c *httpRouteConverter) translate(ctx context.Context, logger logr.Logger)
 				"filterCount", len(rule.Filters))
 
 			for _, filter := range rule.Filters {
-				filterHash := utils.Hash32(filter)
-				pluginName := namegen.NewName(httpRouteName, cpRefName, filterHash).String()
+				pluginName := namegen.NewKongPluginName(filter)
```

Comment:
Since this modification does not include information related to route or cp, I would like to know if this is what we expected? Also, will debugging become more complicated after this change?

If possible, I suggest adding some comments here for clarification.